### ISSUE TITLE
Templates for `pubs note` if it is the first time to edit

### ISSUE THREAD

**Issue by m15a** [pubs/pubs#274]
When creating a new note by `pubs note`, its content is blank. It would be helpful for note taking if some templates are available.

**Comment by zamlz** [comment]
You can emulate this by setting the application that you wish to note take
with as a script that emulates this behaviour.
Instead of setting the editor to vim, I had a wrapper script that would
check to see if the file existed first and if it did, open it in vim.
else it would copy a template file to the location, and then open it in vim.

**Comment by m15a** [comment]
> Instead of setting the editor to vim, I had a wrapper script that would check to see if the file existed first and if it did, open it in vim. else it would copy a template file to the location, and then open it in vim.

That's a nice trick, thanks! Is it possible to create a plugin that provides this functionality?

**Comment by zamlz** [comment]
Yeah that's possible. I don't know if I have time to work on it at the
moment, but I know it shouldn't be too complicated.

**Comment by sevehub** [comment]
Here's a similar workaround for VIM.
https://tinyurl.com/y8x84m4o
